test/ui/browser/test-browser-util.js

```diff
@@ -6,32 +6,75 @@ import { fixURL, getSearchURL } from '../../../ui/browser/browser-util';
 
 describe('utils', () => {
   describe('fixURL', () => {
-    it('should leave URIs unchanged', () => {
-      const UNCHANGED_URIS = [
+    it('should leave URLs unchanged', () => {
+      const UNCHANGED_URLS = [
         'data:text/html,',
         'http://example.com',
+        'http://foo.com',
+        'https://foo.com',
+        'https://foo.com/?a=%20b',
+        'data:about',
+        'view-source:http://foo.com/',
+        'rtsp://100.100.100.100/rtsp.mp4',
+
+        // @TODO Handle spaces in GET parameters
```

Comment:
For tests, we should add pending tests -- which is `it()` with only one string argument, which will show up pending